README.md

```diff
@@ -54,6 +54,24 @@ jaf -configFile example.conf
 ```
 Of course, you can also write a init system script to handle this for you.
 
+### Running from Docker
+Running it from the GitHub Container Registry
+```bash
+docker run \
+    -v /path/to/your/config.conf:/app/jaf.conf \
+    -v /path/to/where/you/want/your/files/to/be/stored:/FileDir/in/your/config \
+    ghcr.io/leon-richardt/jaf:latest
+```
+
+Building the Docker image and running it locally
+```bash
+docker build -t jaf .
+docker run \
+    -v /path/to/your/config.conf:/app/jaf.conf \
+    -v /path/to/where/you/want/your/files/to/be/stored:/FileDir/in/your/config \
+    jaf
+```
```

Comment:
Since we're providing a (or rather, using the) default value for the config path, I'm wondering whether it would make sense to set up a reasonable default for `FileDir` as well. One way could be creating and setting the directory in the Dockerfile but maybe there are nicer alternatives too.

What is your take on this?

Reply:
I would be okay with the Dockerfile creating the `/var/www/jaf.example.com/` directory to make the initial setup easier. This would also mean updating the documentation to replace `-v /path/to/where/you/want/your/files/to/be/stored:/FileDir/in/your/config` with `-v /path/to/where/you/want/your/files/to/be/stored:/var/www/jaf.example.com/`
Is this what you mean?

Reply:
That's precisely what I meant. I'm only still debating whether the `/var/www/jaf.example.com/` path would serve as a good default or if maybe something like `/var/www/jaf/` would be nicer

Reply:
Resolved via #2.